app/templates/components/modals/proposal-delete-modal.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="header">
+  {{t 'Are you sure you would like to delete this proposal?'}}
+  <div class="muted small text">
+    {{t 'Deleting the proposal will delete all the data associated with it.'}}
+  </div>
+</div>
+<div class="content">
+  <form class="ui {{if isLoading 'loading'}} form" id="delete-form" autocomplete="off" {{action deleteProposal on='submit' preventDefault=true}}>
+    <div class="field">
+      <div class="label">
+        {{t 'Proposal name you are deleting:'}}
+      </div>
+      {{input type='text' name='confirm_name' value=proposalName disabled=true}}
+    </div>
+  </form>
+</div>
+<div class="actions">
+  <button type="button" class="ui black button" {{action 'close'}}>
+    {{t 'Cancel'}}
+  </button>
+  <button type="submit" form="delete-form" class="ui red button">
```

Comment:
Shouldn't the button be disabled unless the value in the text field matches the name of the session?